[2024-10-06T16:08:49.284+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-06T16:08:49.917+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.extract_csv manual__2024-10-06T23:07:58+07:00 [queued]>
[2024-10-06T16:08:49.953+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.extract_csv manual__2024-10-06T23:07:58+07:00 [queued]>
[2024-10-06T16:08:49.958+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-06T16:08:50.007+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_csv> on 2024-10-06 16:07:58+00:00
[2024-10-06T16:08:50.033+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=138) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-06T16:08:50.043+0000] {standard_task_runner.py:72} INFO - Started process 141 to run task
[2024-10-06T16:08:50.063+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'extract_csv', 'manual__2024-10-06T23:07:58+07:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpsg8hyf3p']
[2024-10-06T16:08:50.083+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask extract_csv
[2024-10-06T16:08:50.318+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.extract_csv manual__2024-10-06T23:07:58+07:00 [running]> on host 8dc237c78455
[2024-10-06T16:08:50.766+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-10-06T16:07:58+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-06T23:07:58+07:00'
[2024-10-06T16:08:50.777+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-06T16:08:51.152+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-06T16:08:51.213+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-06T16:08:51.227+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=extract_csv, run_id=manual__2024-10-06T23:07:58+07:00, execution_date=20241006T160758, start_date=20241006T160849, end_date=20241006T160851
[2024-10-06T16:08:51.323+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-06T16:08:51.414+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-06T16:08:51.418+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
